bus-seat/src/main/java/org/kernel360/busseat/BusSeatApplication.java

```diff
@@ -2,8 +2,10 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
 
 @SpringBootApplication
+@EnableJpaAuditing
```

Comment:
```suggestion
@EnableJpaAuditing
```

생성일, 수정일 공통처리를 위한 어노테이션 제거해야함